dspace/config/dspace.cfg

```diff
@@ -1972,6 +1963,9 @@ mail.helpdesk = ${mail.admin}
 # Should all Request Copy emails go to the helpdesk instead of the item submitter?
 request.item.helpdesk.override = false
 
+# 2 MB
```

Comment:
can you add more details about the intend of the parameter? it is a limit for the range requests? it is discoverable in some way?

Reply:
This is the size of the byte buffer [used for copying data from the bitstream FileInputStream to the request OutputStream](https://github.com/atmire/DSpace/blob/ae0d6770f9e5ef03ec0d683fdcf1d6eab33e927b/dspace-spring-rest/src/main/java/org/dspace/app/rest/utils/MultipartFileSender.java#L430). We can also omit the configuration property and give it a static value.